web/src/routes/OrganizationsList/Organization/Tabs/Settings/GeneralSettings.tsx

```diff
@@ -0,0 +1,341 @@
+    timeMachineOptions[quayConfig?.config.TAG_EXPIRATION_OPTIONS[0]],
+  );
+  const namespaceTimeMachineExpiry = isUserOrganization
+    ? user?.tag_expiration_s
+    : (organization as IOrganization)?.tag_expiration_s;
+
+  // Email
+  const namespaceEmail = isUserOrganization
+    ? user?.email || ''
+    : (organization as any)?.email || '';
+  const [emailFormValue, setEmailFormValue] = useState<string>('');
+  const [fullNameValue, setFullNameValue] = useState<string>(null);
+  const [companyValue, setCompanyValue] = useState<string>(null);
+  const [locationValue, setLocationValue] = useState<string>(null);
+  const [validated, setValidated] = useState<validate>('default');
+
+  useEffect(() => {
+    setEmailFormValue(namespaceEmail);
+    setFullNameValue(user?.family_name || null);
+    setCompanyValue(user?.company || null);
+    setLocationValue(user?.location || null);
+    const humanized_expiry = humanizeTimeForExpiry(namespaceTimeMachineExpiry);
+    for (const key of Object.keys(timeMachineOptions)) {
+      if (humanized_expiry == timeMachineOptions[key]) {
+        setTimeMachineFormValue(key);
+        break;
+      }
+    }
+  }, [loading, isUserLoading, isUserOrganization]);
+
+  const handleEmailChange = (emailFormValue: string) => {
+    setEmailFormValue(emailFormValue);
+    if (namespaceEmail == emailFormValue) {
+      setValidated('default');
+      setError('');
+      return;
+    }
+
+    if (!emailFormValue) {
+      setValidated('error');
+      setError('Please enter email associate with namespace');
+      return;
+    }
+
+    if (namespaceEmail != emailFormValue) {
+      if (isValidEmail(emailFormValue)) {
+        setValidated('success');
+        setError('');
+      } else {
+        setValidated('error');
+        setError('Please enter a valid email address');
+      }
+    }
+  };
+
+  const checkForChanges = () => {
+    if (!isUserOrganization && namespaceEmail != emailFormValue) {
+      return validated == 'success';
+    }
+
+    console.log(user);
+    console.log(emailFormValue, fullNameValue, companyValue, locationValue);
+    return (
+      (getSeconds(timeMachineFormValue) != namespaceTimeMachineExpiry ||
+        namespaceEmail != emailFormValue ||
+        user?.family_name != fullNameValue ||
+        user?.company != companyValue ||
+        user?.location != locationValue) &&
+      validated != 'error'
+    );
+  };
+
+  const updateSettings = async () => {
+    try {
+      if (!isUserOrganization) {
+        const response = await updateOrgSettings({
+          tag_expiration_s:
+            getSeconds(timeMachineFormValue) != namespaceTimeMachineExpiry
+              ? getSeconds(timeMachineFormValue)
+              : null,
+          email: namespaceEmail != emailFormValue ? emailFormValue : null,
+          isUser: isUserOrganization,
+        });
+        return response;
+      } else {
+        const response = await updateUser({
```

Comment:
Is this missing time machine settings for the user? Currently setting the time machine for user namespaces doesn't do anything. Org namespaces still work.